benchmark_test.go

```diff
@@ -86,7 +86,7 @@ func (suite *BenchmarkSuite) BenchmarkSingleCon1k(c *C) {
 //benchmark single consumer multi 100 1k payload
 func (suite *BenchmarkSuite) BenchmarkSingleConMulti1k(c *C) {
 	for i := 0; i < c.N; i++ {
-		p, _ := suite.consumer1k.MultiGet(100)
+		p, _ := suite.consumer1k.MultiGet(100, 0)
 		p[99].MultiAck()
```

Comment:
We don't follow the convention of 0 * time.Second here for the timeout? This gets casted automatically to the expected type?